diff -Naur make-4.1.orig/doc/Makefile.in make-4.1/doc/Makefile.in
--- make-4.1.orig/doc/Makefile.in	2014-10-05 18:25:22.000000000 +0200
+++ make-4.1/doc/Makefile.in	2015-01-22 06:29:09.153092475 +0100
@@ -174,7 +174,6 @@
     n|no|NO) false;; \
     *) (install-info --version) >/dev/null 2>&1;; \
   esac
-am__installdirs = "$(DESTDIR)$(infodir)"
 am__vpath_adj_setup = srcdirstrip=`echo "$(srcdir)" | sed 's|.|.|g'`;
 am__vpath_adj = case $$p in \
     $(srcdir)/*) f=`echo "$$p" | sed "s|^$$srcdirstrip/||"`;; \
@@ -575,11 +574,8 @@
 	  dist-info
 check-am: all-am
 check: check-am
-all-am: Makefile $(INFO_DEPS)
+all-am: Makefile
 installdirs:
-	for dir in "$(DESTDIR)$(infodir)"; do \
-	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
-	done
 install: install-am
 install-exec: install-exec-am
 install-data: install-data-am
@@ -631,7 +627,7 @@
 
 info-am: $(INFO_DEPS)
 
-install-data-am: install-info-am
+install-data-am:
 
 install-dvi: install-dvi-am
 
